test/org/sagebionetworks/bridge/services/ConsentServiceMockTest.java

```diff
@@ -88,116 +106,99 @@
     @Mock
     private SubpopulationService subpopService;
     @Mock
+    private NotificationsService notificationsService;
+    @Mock
+    private S3Helper s3Helper;
+    @Mock
+    private UrlShortenerService urlShortenerService; 
+    @Mock
     private Subpopulation subpopulation;
+    @Mock
+    private StudyConsentView studyConsentView;
     @Captor
     private ArgumentCaptor<BasicEmailProvider> emailCaptor;
     @Captor
     private ArgumentCaptor<WithdrawConsentEmailProvider> withdrawalProviderCaptor;
+    @Captor
+    private ArgumentCaptor<SmsMessageProvider> smsProviderCaptor;
+    @Captor
+    private ArgumentCaptor<Account> accountCaptor;
     
     private String documentString;
-        
-    private CriteriaContext context;
-    private Study study;
-    private StudyParticipant participant;
-    private ConsentSignature consentSignature;
+
     private Account account;
     
     @Before
     public void before() throws IOException {
         documentString = IOUtils.toString(new FileInputStream("conf/study-defaults/consent-page.xhtml"));
-                
+        account = new GenericAccount();
```

Comment:
We should create Account.create() instead of leaving a GenericAccount reference here.

Reply:
I've done this in the other PR that's currently pending, so I will resolve this on merge. I could change it here too but it'll lead to more merge conflicts.